RosNet/Exceptions/MissingHeaderFieldException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace RosNet.Exceptions;
```

Comment:
Can this please not be in it's own namespace, that is just inpractical for every user, just place exceptions in the root namespace

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces for further information about how namespaces should be used